### DISCUSSION TITLE
关于标题与正文间的垂直距离问题

### DISCUSSION THREAD

**Discussion by Linkzhuu** [nju-lug/NJUThesis#110] [Q&A] [answered]
又时候标题和正文之间会空出很大一份空白
<img width="456" alt="截屏2022-03-28 上午1 03 14" src="https://user-images.githubusercontent.com/47322049/160292544-23ee32f4-1902-4bf3-b465-f423f999820a.png">

**Comment by atxy-blip** [comment, marked as answer]
这是弹性间距，很可能是因为前后图表比较多而文字比较少。建议等完稿后统一调整图片大小、图表的定位方式（`[htbp]`）。
